galley/pkg/config/processor/metadata/metadata.yaml

```diff
@@ -555,11 +554,30 @@ snapshots:
       - "istio/config/v1alpha2/legacy/tracespans"
       - "istio/config/v1alpha2/legacy/zipkins"
 
+    # Used by Galley to perform service discovery
   - name: "syntheticServiceEntry"
     strategy: immediate
     collections:
       - "istio/networking/v1alpha3/synthetic/serviceentries"
 
+    # Used by istioctl to perform analysis
+  - name: "localAnalysis"
```

Comment:
This may be a followup task (and if so, I can take it on) but we should update https://github.com/istio/istio/blob/master/galley/pkg/config/analysis/README.md to be clear that analyzers will only run if the all the collections they use appear in either the localAnalysis or syntheticServiceEntry snapshots here.